ccm_web/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "dev:server": "nodemon --config nmon_config.json server/app.ts",
+    "dev:server": "nodemon --inspect=0.0.0.0 --config nmon_config.json server/app.ts",
```

Comment:
Is this where you'd put the port? I'd probably make this explicit like

-inspect=0.0.0.0:9229